draft-ietf-quic-multipath.md

```diff
@@ -658,7 +658,9 @@ with Path ID 0.
 The PATH_NEW_CONNECTION_ID frame is used to issue new connection IDs for all paths.
 In order to let the peer open new paths, it is RECOMMENDED to proactively
 issue a Connection ID for at least one unused Path ID, as long as it remains
-compatible with the peer's Maximum Path ID limit.
+compatible with the peer's Maximum Path ID limit. An endpoint MAY issue at
```

Comment:
Do we need normative MAY here?

Reply:
We don't. I just thought because we have a normative RECOMMENDED in the previous sentence this would make sense here...?